### DISCUSSION TITLE
우리만의 기술 블로그를 만들어볼까?

### DISCUSSION THREAD

**Discussion by verus-j** [woowacourse-teams/2022-moamoa#73] [공유]
프로젝트를 진행하면서 겪은 트러블슈팅이나 경험을 작성하는 간단한 모아모아의 기술 블로그를 만들어보는 건 어떨까??

**Comment by sc0116** [comment]
헉 한번 해보고싶네

**Comment by tco0427** [comment]
다른 팀처럼 돌아가면서 하나씩 주제를 선정(말씀하셨던 것 처럼 프로젝트에서 겪은 경험 or 트러블슈팅도 좋음)해서 작성하는 것도 나쁘지 않다고 생각합니다! (물론 현재 팀 작업 이외의 추가적인 작업이라 개개인의 시간 투자가 필요해 부담스러울 수 있음)

**Comment by airman5573** [comment]
좋은 생각입니다!

**Comment by verus-j** [comment]
흠... 한번 달려봐?? ㅋㅋㅋㅋ
이제 절반정도 남았지만 먼가 생긴 트러블슈팅 관련 글을 작성하면 좋을 듯? 일기 같은 느낌도 좋고 ㅎㅎ
특히, 이번에 SWAP 메모리를 활용해 NPM 빌드 성공한 사례나 CD를 위해서 우리만의 방식을 도입한 얘기도 괜찮고
나중에 이력서 쓸 때 보여줄 것들이 많아 질듯?

**Reply by tco0427** [reply]
동의합니다..!! 이미 해결했던 것들도 기억을 더듬어보면서 함께 정리하면 더 좋을 것 같습니다!! (개인 블로그에 정리한 것들 등등)
